main.nf

```diff
@@ -623,7 +625,7 @@ process dupradar {
     script: // This script is bundled with the pipeline, in NGI-RNAseq/bin/
     def paired = single ? 'FALSE' :  'TRUE'
     """
-    dupRadar.r $bam_md $gtf $paired
+    dupRadar.r $bam_md $gtf $paired ${params.rlocation}
```

Comment:
well it ran fine on Travis so I guess it didn't break. But I'll try and test it properly myself